docs/getting-started-guides/gce.md

```diff
@@ -44,7 +44,7 @@ The next few steps will show you:
 
 ### Installing the kubernetes client on your workstation
```

Comment:
I don't like `kubernetes client` phrase. I'd rather have `kubernetes command line tools` or something which does not collide with internal `client` library.